en-GB/Settings.js

```diff
@@ -57,7 +57,7 @@ export default {
       'Description': 'Stop & Start the bot'
     },
     'TwitchBotCommands': {
-      'Title': 'Customise the Bot Command names',
+      'Title': 'Customise Bot Command',
```

Comment:
shouldn't this be like `Commands` because these are plural? because it was `names` before that

Reply:
Agreed on that! 😄

Reply:
Oh God, how did I miss this?!